libpy/durable/lang.py

```diff
@@ -201,7 +203,7 @@ def define(self):
             if self._type == '$s':
                 raise Exception('s not allowed as rvalue')
 
-            print('defining {0}, {1}'.format(self._type, self._left))
+            logging.debug('defining {0}, {1}'.format(self._type, self._left))
```

Comment:
Use the parameters of `logging.debug`
```python
logging.debug('defining %s, %s', self._type, self._left)
```

Reply:
Removed all calls to `format` in logging calls